### DISCUSSION TITLE
Modifying predefined BricsCAD block attributes

### DISCUSSION THREAD

**Discussion by daalgi** [mozman/ezdxf#1119] [Q&A]
Hello,

I was testing this amazing library, but I've been stuck for a while on something.

I have a dxf (R2018) file with a block created in BricsCAD. The block consists of a rectangle, and a text element centered and sized according to that rectangle. The text value is defined as an attribute called "tag", allowing me to change the block shown text by simply modifying the attribute value.

Now, I'm trying to open that dxf file with `ezdxf` and insert an instance of that block into the model space. However, I'm having trouble accessing the  predifined "tag" attribue. While I can add or overwrite the "tag" attribute with `block.add_attrib("tag", "8")`, this approach results in the loss of the predefined text format.

Is there any way to access and modify the value of predefined block attributes without losing the original format?

**Comment by mozman** [comment]
https://ezdxf.mozman.at/docs/howto/content.html#get-set-block-reference-attributes
